ForkClone.md

```diff
@@ -1,33 +1,33 @@
-# Fork and Clone 
+# Fork and clone 
 
-## Steps to fork the repo: 
+## Steps to fork the repository (repo): 
```

Comment:
I noticed we go back and forth between "repository" and "repo." I chose to spell it out on first use with an abbreviation in parentheses and then use the shortened version thereafter, which many style guides prefer. However, we could always use the long version or always use the short version if you prefer. As long as we're consistent, it's a style decision.

Reply:
Also, many style guides say you shouldn't have colons in headings. However, some tech say you must always introduce numbered steps with a colon. If we want to make everyone happy, we could add a sentence after each heading introducing the numbered steps. So something like: 

## Heading 

To do something, do:

1. Numbered step
1. Numbered step